.github/workflows/build-examples-gh-pages-on-push.yml

```diff
@@ -0,0 +1,31 @@
+name: "ESP-IDF build examples to github pages (push)"
+
+on:
+    push:
+        branches:
+        - master
+
+jobs:
+
+  build-and-upload-binaries:
+    strategy:
+      matrix:
+        idf_ver: ["latest"]
```

Comment:
I'm a little worried about the esp-idf version here changing without our control.

Since we don't have a 'stable' tag, I'd go with more defensive latest release (v5.1.1) and we can update this manually to avoid regressions

Reply:
I have updated the version in the matrix, it should be correct now